frontend/layouts/GatedLayout/_styles.scss

```diff
@@ -2,4 +2,11 @@
     .flash-message {
         border: 0;
     }
+    a {
```

Comment:
Are you sure this selector is ok?  Its not very specific and all images within an anchor on the pages wrapped by `GatedLayout` (e.g. `RegistrationPage`, `LoginPreviewPage`,`LoginPage`, `ConfirmInvitePage`, `COnfirmSSOIncitePage`, `ForgotPasswordPage`. and `ResetPasswordPage`) will have these styles applied.

We could be more specific and add a class specifically to where this markup is and target the styles to that class. I'm guessing this is for the `ForgotPasswordPage` link?